### PR TITLE
User CRUD functions

### PR DESCRIPTION
**Pull request by jillianw**

### CODE REVIEW

**Comment by halfghaninne** [comment]
Can you please give a little more information in your PRs / commit messages? What is not displaying properly? Is the response view loading and the HTML is not what you'd expect? Or is the response not even coming to the browser?

**Comment by jillianw** [comment]
The form is not displaying in the browser for users/new, but if we copy/paste the form to any other view we've created it shows up fine. It's not throwing an error or saying the page doesn't exist. It's just blank as if nothing were in the view file.

**Comment by halfghaninne** [comment]
That seems really strange. Did you inspect the source of the blank page?

**Comment by jillianw** [comment]
It just shows what all is in the layout.erb, not what we've added to our view page.

**Comment by shannonfromomaha** [comment]
RE: the delete function
our error code is "sinatra doesn't know this ditty, try get '/users/1/delete' do"
our form is post
our controller is post
we are stuck as to why it thinks this is a get request.

**Comment by sumeetjain** [comment]
`<form action="users/<%= x.id %>/delete" method "post">` should be:
`<form action="users/<%= x.id %>/delete" method="post">`

**Comment by halfghaninne** [comment]
In which file, Sumeet? I see the `=` in **users/edit.erb**

File: app/views/users/index.erb [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<p>All users:</p>
+
+<% @users.each do |x| %>
+  <p>
+    <a href="users/<%= x.id %>">
+      <%= x.name %>
+    </a>
+  </p>
+  <form action="users/<%= x.id %>/delete" method "post">
```
Comment (sumeetjain) [inline app/views/users/index.erb]: This is where I see the missing `=`.

**Pull request merged**